src/utils/import.ts

```diff
@@ -112,19 +112,23 @@ const selectBroadcastArtifact = async (
       }),
   );
 
-  if (options.length === 0) await exit(`No valid broadcast files found in ${pathToArtifacts}`);
-
-  return (
-    await inquirer.prompt([
-      {
-        type: 'list',
-        name: 'broadcast',
-        message: 'Select a broadcast artifact',
-        // reverse the list so the most recent broadcasts are at the top
-        choices: options.reverse(),
-      },
-    ])
-  ).broadcast;
+  if (options.length === 0) {
+    const error = `No valid broadcast files found in ${pathToArtifacts}`;
+    await exit(error);
+    return error;
```

Comment:
returning the error string is only to adhere to the promised string return. we could alternatively update the promised return to `string | null`

Reply:
`exit()` should have a signature of `Promise<never>` meaning typescript expects execution to halt after it's called. Were you getting errors here?